labo4ka5.py

```diff
@@ -0,0 +1,54 @@
+from enum import Enum
+
+class Gender(Enum):
+    FEMALE = 1
+    MALE = 2
+    NON_BINARY = 3
+
+class Guest:
+    def __init__(self, guest_id, name, age, city, phone_number, gender):
+        self.id = guest_id
+        self.name = name
+        self.age = age
+        self.city = city
+        self.phone_number = phone_number
+        self.gender = gender
+
+class Party:
+    def __init__(self, day, reason, guests):
+        self.day = day
+        self.reason = reason
+        self.guests = guests
+
+    def FindAverageAge(self, gender):

```

Comment:
Naming Convention Violation (PEP8)
```suggestion
    def find_average_age(self, gender):

```